README.md

```diff
@@ -109,3 +103,26 @@ aws_secret_access_key = <your secret access key id>
 ```
 export AWS_PROFILE=tsl
 ```
+
+## Deployment
+This app is deployed via Heroku. To run the deploy commands, please [install Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli).
+
+
+### Staging environment setup
+
+#### Initializing staging database
+*This is a temporary workaround* First, replace the values for user, password, host, and database in `src/server/database.json`. Once those values have been added, run: 
```

Comment:
Wanted to highlight this temporary workaround for setting up the staging database.

Reply:
Thanks for highlighting! I'm glad you pointed that out, I'll add a task to set up some env variables handling. I think having a programmatic way for determining what those are will solve this and a couple other issues (needing to export variables on the command line, having a secret committed to our repo).

Reply:
Oh wait, looks like we're using `dotenv`? That might get us the tools we need to solve this issue in the long term.